### PR TITLE
143| Sree Ranjani/Nazeer | Adds Base64 encode for the auth code

### PR DESCRIPTION
**Pull request by SreeRanjani14**

### CODE REVIEW

File: src/In.ProjectEKA.HipService/UserAuth/UserAuthService.cs [hunk 1]
```diff
@@ -12,10 +14,12 @@ namespace In.ProjectEKA.HipService.UserAuth
     public class UserAuthService : IUserAuthService
     {
         private readonly IUserAuthRepository userAuthRepository;
+        private readonly ILogger<UserAuthController> logger;
 
-        public UserAuthService(IUserAuthRepository userAuthRepository)
+        public UserAuthService(IUserAuthRepository userAuthRepository, ILogger<UserAuthController> logger)
         {
             this.userAuthRepository = userAuthRepository;
+            this.logger = logger;
```
Comment (praveenadayanand) [inline src/In.ProjectEKA.HipService/UserAuth/UserAuthService.cs:22]: is logger used anywhere in this code?

**Review by praveenadayanand — CHANGES_REQUESTED** [review]

**Review by swatigogia2020 — APPROVED** [review]
Remove the logger if not used

**Review by praveenadayanand — APPROVED** [review]

**Review by MeghnaThoughtworks — CHANGES_REQUESTED** [review]
Please add tests for new code

**Comment by Nazeer-debug** [comment]
> Please add tests for new code

Tests for the new method or auth confirm flow?

**Review by MeghnaThoughtworks — APPROVED** [review]

**Pull request merged**